src/main/java/com/unascribed/fabrication/mixin/b_utility/despawning_items_blink/MixinItemEntityRenderer.java

```diff
@@ -25,7 +25,7 @@ public void render(ItemEntity entity, float yaw, float tickDelta, MatrixStack ma
 			RenderingAgeAccess aa = (RenderingAgeAccess)entity;
 			int age = aa.fabrication$getRenderingAge();
 			int timeUntilDespawn = 6000-age;
-			if (timeUntilDespawn < 40) {
+			if (timeUntilDespawn < 20) {
```

Comment:
Excuse me? This is a functional change.

Reply:
The comment said 1 sec(20)
and the condition undearneth is supposed to be 2sec
so i just assumed this is a mistake

Reply:
It probably originally was 1 second but I changed it while doing visual tweaking. Guess I didn't update the comment.